lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -1137,15 +1137,17 @@ def create_testing_determination_internal_questionnaire_response(record: dict, p
     questionnaire we do want to return answers for all questions even if they are
     not answered.
     """
+    boolean_questions = [
+        'testing_trigger',
+    ]
 
     string_questions = [
-        'testing_trigger',
         'testing_type',
```

Comment:
It looks like testing_type turned out to have numeric values instead of text. You could make this an integer question if you wanted.

Reply:
Hmm, we should be ingesting the label of testing_type, not the raw integer code for it.

Reply:
These don't have human readable codes for them, so we would have to map the integer code to the labels.
(Sorry, I must have missed this when I asked them to update codings)

Reply:
How can you tell @joverlee521 ? In REDCap, the data dictionary looks like this: 
![image](https://user-images.githubusercontent.com/11384464/95915070-7c170300-0d5b-11eb-8f85-09884e513cda.png)

Reply:
The first column is the coding that we pull in for the ETL. Compare to other fields that do have human readable codes:

![Screen Shot 2020-10-13 at 1 54 18 PM](https://user-images.githubusercontent.com/40774762/95915331-d87a2280-0d5b-11eb-9ece-014af1d80613.png)

Reply:
Got it, thanks.

Reply:
Ah, in that case I'm going to leave `testing_type` as a string for now because we'll probably want to convert it to a human readable code at some point. Does that sound OK?

Reply:
Yeah, not even sure we truly need to ingest `testing_type`. I'm not currently using this field in the quota query.

Reply:
The quota system should set a value in testing_type. If we ingest it it could compare to what's already there.

Reply:
I'll leave it alone for now.